cluster-provision/k8s/1.19/bind_device_to_vfio.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
yeah, I would love to move it to a common location, but I see all of the scripts are being duplicated between providers, so I'm not sure ho...

Reply:
This is done in purpose so every provider is isolated, we have suffered already for a bash scripts dependency tree and since we usuaslly has three providers kind of active is not that much to change it three times if we have too.

Reply:
Please no common locations. Also in general not all versions need to  be updated with new features. Very often it is not even possible to enable the same feature for all providers.

Reply:
In this case it would perfectly fine to only add this to 1.18 and 1.19, or even 1.19 only.

Reply:
Also it is fine if gocli can't use some featurs on some clusters.

Reply:
> In this case it would perfectly fine to only add this to 1.18 and 1.19, or even 1.19 only.

@rmohr it's not in a common location. I've added this per provider. We need to enable this on the 3 latest providers so kubevirt tests will pass on all 3 of them.

Reply:
@vladikr you can always skip some tests on some versions (up to you of course). I know that it is not  on a common place. I just wanted to highlight that we really don't want a common place :)

Reply:
noted :-)